<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginPageClass</class>
 <widget class="QMainWindow" name="LoginPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginPage</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(&quot;:/GymPadelSystem/Resources/Gym.png&quot;);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="LWidget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>320</width>
      <height>580</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: none;
background-color: rgba(0,0,0,0.7);
border-radius: 25px;</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>320</width>
       <height>70</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
font-size: 30px;
color: rgba(135, 206, 235,0.8)</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="FNTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;
border-color:  rgba(135, 206, 235,0.8);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MNTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Middle Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="LNTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDTextBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>260</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
background-color: rgba(250,250,250,0.7);
padding-left: 15px;
font-size: 15px;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="SMLabel">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>420</y>
       <width>220</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
font-size: 15px;
background-color: rgba(250, 0, 0, 0.5);
border-radius: 20px;</string>
     </property>
     <property name="text">
      <string>Invalid Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="LoginButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>121</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background: none;
background-color: rgba(135, 206, 235,0.9);
color: black;
border-radius: 25px;
}
QPushButton:hover
{
border: 3px solid rgba(135, 206, 235,0.9);
background-color: rgba(250, 250, 250,0.8);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SignUpButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>510</y>
       <width>121</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background: none;
background-color: rgba(135, 206, 235,0.9);
color: black;
border-radius: 25px;
}
QPushButton:hover
{
border: 3px solid rgba(135, 206, 235,0.9);
background-color: rgba(250, 250, 250,0.8);
}</string>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="NewHereLabel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>480</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
text-decoration: underline;
color: rgba(135, 206, 235,0.8)</string>
     </property>
     <property name="text">
      <string>New here?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>1420</x>
      <y>800</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background: none;
background-color: rgba(255, 51, 51,0.9);
color: white;
border-radius: 25px;
}
QPushButton:hover
{
border: 2px solid rgba(255, 51, 51,0.9);
background-color: rgba(250, 250, 250,0.8);
color: rgb(255, 51, 51);
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
